### PR TITLE
Variables renaming and minor adjusments

### PR DESCRIPTION
**Pull request by irosenzw**

### CODE REVIEW

**Comment by mkopta** [comment]
Seems like it needs to be rebased, right?

**Comment by eedri** [comment]
ci test please

**Comment by david-caro** [comment]
ci test please

**Comment by david-caro** [comment]
This one needs some rework, it does not pass the yapf check:

```
14:09:01 -------------------------------------------------------------
14:09:01 -------------------------------------------------------------
14:09:01 -~      Running style checks                               --
14:09:01 -------------------------------------------------------------
14:09:01 scripts/check_style.sh
14:09:01 yapf 0.7.1
14:11:08 --- lago/prefix.py	(original)
14:11:08 +++ lago/prefix.py	(reformatted)
14:11:08 @@ -106,12 +106,11 @@
14:11:08          self._metadata = None
14:11:08          self._internal_repo = None
14:11:08  
14:11:08 -    def _get_internal_repo (self):
14:11:08 +    def _get_internal_repo(self):
14:11:08          return self._internal_repo
14:11:08  
14:11:08 -    def _set_internal_repo (self, path):
14:11:08 +    def _set_internal_repo(self, path):
14:11:08          self._internal_repo = path
14:11:08 -
14:11:08  
14:11:08      def _get_metadata(self):
14:11:08          """
14:11:08 --- ovirtlago/__init__.py	(original)
14:11:08 +++ ovirtlago/__init__.py	(reformatted)
14:11:08 @@ -216,13 +216,11 @@
14:11:08          if not projects_list:
14:11:08              projects_list = PROJECTS_LIST
14:11:08  
14:11:08 -
14:11:08          custom_sources = custom_sources or []
14:11:08  
14:11:08          rpm_dirs = []
14:11:08  
14:11:08          for dist in dists:
14:11:08 -
14:11:08  
14:11:08              project_roots = [
14:11:08                  self.paths.build_dir(project_name)
14:11:08 --- ovirtlago/cmd.py	(original)
14:11:08 +++ ovirtlago/cmd.py	(reformatted)
14:11:08 @@ -152,7 +152,8 @@
14:11:08  @in_ovirt_prefix
14:11:08  @with_logging
14:11:08  def do_ovirt_reposetup(
14:11:08 -    prefix, rpm_repo, reposync_yum_config, skip_sync, custom_sources, internal_repo_path, **kwargs
14:11:08 +    prefix, rpm_repo, reposync_yum_config, skip_sync, custom_sources,
14:11:08 +    internal_repo_path, **kwargs
14:11:08  ):
14:11:08  
14:11:08      if kwargs['engine_dir']:
14:11:08 Yapf failed, make sure to run:
14:11:08     yapf --style .style.yapf --in-place --recursive .
14:11:08 
14:11:08 to format any python files automatically, it is higly recommended that you
14:11:08 install formatting tools to your editor, check the yapf repo:
14:11:08 
14:11:08      https://github.com/google/yapf/tree/master/plugins
14:11:08 
14:11:08 to check for official support (always helpful).
14:11:08 You can also install the pre-commit hook provided in this repo:
14:11:08 
14:11:08     ln -s scripts/pre-commit.style .git/pre-commit
14:11:08 
14:11:08 and it will format any files changed at commit time.
```

**Comment by david-caro** [comment]
this branch now includes some extra functionality that is also on the pr #291, can you remove that last commit from here? (or merge both prs into one if the variable name adjustments make sense into the other pr).

Thanks!!

File: ovirtlago/cmd.py [hunk 1]
```diff
@@ -174,13 +179,20 @@ def do_ovirt_reposetup(
...
 @cli_plugin(help='Run scripts that install necessary RPMs and configuration')
+@cli_plugin_add_argument(
+    '--internal-repo',
+    help='Path to local rpm repository',
+    type=os.path.abspath,
+)
 @in_ovirt_prefix
 @with_logging
 def do_deploy(prefix, **kwargs):
+    prefix._set_internal_repo(kwargs['internal_repo'])
```
Comment (david-caro) [inline ovirtlago/cmd.py]: Hmmm... I think that this might better be done in the constructor of the prefix instead of here (check the in_prefix function)

File: lago/prefix.py [hunk 2]
```diff
@@ -104,6 +104,14 @@ def __init__(self, prefix):
...
         self._virt_env = None
         self._metadata = None
+        self._internal_repo = None
+
+    def _get_internal_repo (self):
+        return self._internal_repo
+
+    def _set_internal_repo (self, path):
+        self._internal_repo = path
+
```
Comment (david-caro) [inline lago/prefix.py]: this setter and getter are not needed, you can just use instance._internal_repo to access the property

File: lago/plugins/vm.py [hunk 3]
```diff
@@ -360,7 +360,7 @@ def wait_for_ssh(self):
         return ssh.wait_for_ssh(
             ip_addr=self.ip(),
             host_name=self.name(),
-            connect_retries=self._spec.get('boot_time_sec', 50),
+            connect_timeout=self._spec.get('boot_time_sec', 600),
```
Comment (david-caro) [inline lago/plugins/vm.py]: this seems unrelated to the pr

Comment (david-caro) [inline lago/plugins/vm.py]: Though I completely agree that 'boot_time_sec' was totally misleading

Comment (david-caro) [inline lago/plugins/vm.py]: To tweak the ssh timeout you have the ssh_timeout conf value:

https://github.com/lago-project/lago/pull/287/files#diff-031917605b8ea7e9f49d56da61e00f6eR319

Not sure if that's what you wanted here.

Comment (irosenzw) [inline lago/plugins/vm.py]: The idea here isn't about making the ssh timeout longer. (I'm familiar with
the conf file )
As I was written in the PR, when the host in is booting and got it's IP and
didn't open port 22 or get sshd up and running - it will block any ssh
communication from Lago . so, no matter if I will raise the ssh timeout to
1M secs , Lago won't wait for the ssh connection the time that configured -
it will stop waiting and try again. In this situation we loose 1 retry in
one second instead of the time configured (10 sec at default).

My suggestion is ,wait for the connection to the host a specific amount of
time (say 10 minutes), at that time, Lago will try to connect to the host
(with an interval of 1 sec like now) as many times that it can.
